terraform/modules/grafana/subnets.tf

```diff
@@ -0,0 +1,63 @@
+data "aws_availability_zones" "available" {}
+
+data "aws_nat_gateway" "public" {
+  count = var.az_count
+  tags = {
+    Name = "nat-gateway-${count.index}-mgmt-mgmt"
+  }
+}
+
+# Bring default security group under Terraform management and remove all rules
+resource "aws_default_security_group" "default" {
+  vpc_id = var.vpc_id
+}
```

Comment:
Do you need this? There's a resource for the default security group already being managed in the Jenkins terraform and there's only one default security group per VPC.

Reply:
removed